From 2ee7e39632a35858066933fe70ebeaf317ba4808 Mon Sep 17 00:00:00 2001
From: Richard Holme <richard.holme@vantiva.com>
Date: Thu, 23 Apr 2026 09:16:23 +0100
Subject: [PATCH] patch 2

---
 src/vendor/vendor_defs.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/vendor/vendor_defs.h b/src/vendor/vendor_defs.h
index b3692fe..e48b2d9 100755
--- a/src/vendor/vendor_defs.h
+++ b/src/vendor/vendor_defs.h
@@ -137,17 +137,17 @@
 
 //-----------------------------------------------------------------------------------------
 // Uncomment the following to remove code and features from the standard build
 //#define REMOVE_DEVICE_INFO               // Removes DeviceInfo from the core data model. It must instead be provided by the vendor.
 //#define REMOVE_DEVICE_TIME               // Removes Device.Time from the core data model. It must instead be provided by the vendor.
 //#define REMOVE_USP_BROKER                // Removes all USP Broker functionality, including Device.USPServices
 //#define REMOVE_USP_SERVICE               // Removes all USP Service functionality
-//#define REMOVE_DEVICE_REBOOT             // Removes Device.Reboot() from the core data model
-//#define REMOVE_DEVICE_FACTORY_RESET      // Removes Device.FactoryReset() from the core data model
-//#define REMOVE_DEVICE_SCHEDULE_TIMER     // Removes Device.ScheduleTimer() from the core data model
-//#define REMOVE_DEVICE_BOOT_EVENT         // Removes Device.Boot! from the core data model
+#define REMOVE_DEVICE_REBOOT             // Removes Device.Reboot() from the core data model
+#define REMOVE_DEVICE_FACTORY_RESET      // Removes Device.FactoryReset() from the core data model
+#define REMOVE_DEVICE_SCHEDULE_TIMER     // Removes Device.ScheduleTimer() from the core data model
+#define REMOVE_DEVICE_BOOT_EVENT         // Removes Device.Boot! from the core data model
 //#define REMOVE_DEVICE_SECURITY           // Removes all certificate and TLS related code. Removes DM elements Device.Security, Device.LocalAgent.Certificate and ControllerTrust Challenge/Response mechanism
 //#define REMOVE_DEVICE_SECURITY_CERTIFICATE    // Removes Device.Security.Certificate and Device.Security from the core data model
-//#define REMOVE_SELF_TEST_DIAG_EXAMPLE    // Removes Self Test diagnostics example code
+#define REMOVE_SELF_TEST_DIAG_EXAMPLE    // Removes Self Test diagnostics example code
 #define REMOVE_IP_CAPACITY_DIAG          // Removes Device.IP.Diagnostics.IPLayerCapacity() from the core data model
 
 //#define DONT_SORT_GET_INSTANCES          // Disables the sorting of data model paths returned in a GetInstancesResponse. Useful for slow devices supporting large data models.
-- 
2.7.4

