diff --git a/src/vendor/vendor_defs.h b/src/vendor/vendor_defs.h
index 28778c5..b027c15 100755
--- a/src/vendor/vendor_defs.h
+++ b/src/vendor/vendor_defs.h
@@ -96,7 +96,7 @@
 
 // Location of the database file to use, if none is specified on the command line when invoking this executable
 // NOTE: As the database needs to be stored persistently, this should be changed to a directory which is not cleared on boot up
-#define DEFAULT_DATABASE_FILE               OBUSPA_LOCAL_STATE_DIR "/usp.db"
+#define DEFAULT_DATABASE_FILE               "/etc/obuspa.db"
 
 // Location of unix domain stream file used for CLI communication between client and server
 #define CLI_UNIX_DOMAIN_FILE                "/tmp/usp_cli"

